src/content/en/updates/2018/07/search-ads-speed.md

```diff
@@ -0,0 +1,73 @@
+
+
+### Speed is now used as a ranking factor for mobile searches
+
+Users want to find answers to their questions quickly and [data](https://www.thinkwithgoogle.com/marketing-resources/data-measurement/mobile-page-speed-new-industry-benchmarks/) shows that people really care about how quickly their pages load. The Search team announced speed would be a [ranking signal](https://webmasters.googleblog.com/2010/04/using-site-speed-in-web-search-ranking.html) for desktop searches in 2010 and as of this month (July 2018), page [speed will be a ranking factor for mobile searches](https://webmasters.googleblog.com/2018/01/using-page-speed-in-mobile-search.html) too. 
+
+If you're a developer working on a site, now is a good time to evaluate your performance using our [speed tools](/web/fundamentals/performance/speed-tools/). Think about how [performance](/web/fundamentals/performance/rail) affects the user experience of your pages and consider measuring a variety of real-world [user-centric performance metrics](/web/updates/2017/06/user-centric-performance-metrics).
+
+Are you shipping [too much JavaScript](/web/fundamentals/performance/optimizing-content-efficiency/javascript-startup-optimization/)? [Too many images](/web/fundamentals/performance/optimizing-content-efficiency/image-optimization)? Images and JavaScript are the [most significant contributors](https://paulcalvano.com/index.php/2018/07/02/impact-of-page-weight-on-load-time/) to the page weight that affect page load time based on data from [HTTP Archive](https://httparchive.org/) and the [Chrome User Experience Report](/web/tools/chrome-user-experience-report/) - our public dataset for key UX metrics as experienced by Chrome users under real-world conditions. 
+
+To evaluate performance, check:
+
+* [PageSpeed Insights](/speed/pagespeed/insights/), an online tool that shows speed [field data](/web/fundamentals/performance/speed-tools/#field_data) for your site, alongside suggestions for common optimizations to improve it. 
+* [Lighthouse](/web/tools/lighthouse/), [a lab tool](/web/fundamentals/performance/speed-tools/#lab_data) providing personalized advice on how to improve your website across performance, accessibility, PWA, SEO, and other best practices.
+
+
+### The Mobile Speed Score for ads landing pages
+
+Advertising and speed go hand in hand, with faster landing pages delivering better ROI. Last week, at Google Marketing Live, the Ads team [introduced the new mobile speed score](https://www.blog.google/products/ads/mobile-landing-page-speed-score/).
+
+<figure>
+	<div class="video-wrapper-full-width">
+		<iframe class="devsite-embedded-youtube-video" data-video-id="MmfaZV96x7A" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen>
```

Comment:
Is there any way to pass a time offset to this embed? The speed bit is ~34:30 in. If not, perhaps we can callout the time offset in the paragraph below?

Reply:
Good question. I wasn't able to find documentation on https://developers.google.com/web/resources/widgets about time offsets being supported. Passing `?t=34` also doesn't appear to be supported.

I'll go ahead and add a note in the caption with a link to the time offset (https://youtu.be/MmfaZV96x7A?t=2055) in case folks want to get directly there.